How to compile notable for arm?

I compiled it by "env GOOS=linux GOARCH=arm GOARM=5 make".
When I run it, it crashed with below output.

panic: could not locate box "static"

goroutine 1 [running]:
github.com/jmcfarlane/notable/vendor/github.com/GeertJohan/go%2erice.MustFindBox(0x40f6a2, 0x6, 0x664ca861)

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -9,6 +9,7 @@
       Please upgrade :)
     * Support for this older encryption mechism will be removed in the
       future
+* Add builds for Arm7 [#95](https://github.com/jmcfarlane/notable/issues/95)
 
 ## v0.1.3 / 2018-09-18
 

--- a/scripts/build.sh
+++ b/scripts/build.sh
@@ -1,20 +1,33 @@
 #!/bin/bash -eux
 
-export GOARCH=amd64
 export buildArch="-X main.buildArch"
 
+arms=(7)
 systems=(darwin freebsd linux windows)
 
-# Compile binaries for the desired operating systems
-for goos in ${systems[@]}; do
-    mkdir -p target/notable-${TAG}.${goos}-amd64
-    cp LICENSE target/notable-${TAG}.${goos}-amd64
-    GOOS=$goos CGO_ENABLED=$CGO_ENABLED go build \
-        -ldflags "$FLAGS $buildArch=${goos}-${GOARCH}" \
-        -o target/notable-${TAG}.${goos}-amd64/notable
+function build () {
+    arch=$1
+    goos=$2
+    label=$3
+
+    mkdir -p target/notable-${TAG}.${goos}-${label}
+    cp LICENSE target/notable-${TAG}.${goos}-${label}
+    GOARCH=$arch GOOS=$goos CGO_ENABLED=$CGO_ENABLED go build \
+        -ldflags "$FLAGS $buildArch=${goos}-${label}" \
+        -o target/notable-${TAG}.${goos}-${label}/notable
     if [ "$goos" == "windows" ]; then
-        mv target/notable-${TAG}.${goos}-amd64/{notable,notable.exe}
+        mv target/notable-${TAG}.${goos}-${label}/{notable,notable.exe}
     fi
+}
+
+# Compile for amd64
+for goos in ${systems[@]}; do
+    build amd64 $goos amd64
+done
+
+# Compile for arm
+for v in ${arms[@]}; do
+    GOARM=$v build arm linux arm${v}
 done
 
 # Macos: create a macos app bundle
@@ -24,8 +37,15 @@ cp target/notable-${TAG}.darwin-amd64/notable \
 
 # Package up the Github release zip files
 pushd target
+
+# Zip amd64
 for goos in ${systems[@]}; do
 	zip -r notable-${TAG}.${goos}-amd64.zip notable-${TAG}.${goos}-amd64
+done
+
+# Zip arm
+for v in ${arms[@]}; do
+    zip -r notable-${TAG}.linux-arm${v}.zip notable-${TAG}.linux-arm${v}
 done
 popd
 

--- a/scripts/release.sh
+++ b/scripts/release.sh
@@ -4,6 +4,9 @@ source /keybase/private/jmcfarlane/env/github.sh
 
 cd $(dirname $0)/..
 
+arms=(7)
+systems=(darwin freebsd linux windows)
+
 # The date is something like: 2017-01-20
 export DATE="$(head -n1 CHANGELOG.md | grep -E -o '[0-9]{4}-[0-9]{2}-[0-9]{2}')"
 
@@ -29,8 +32,8 @@ github-release release \
     --description "$DESC" \
     --pre-release
 
-# Upload zip files
-for goos in darwin freebsd linux windows; do
+# Uploads for amd64
+for goos in ${systems[@]}; do
     github-release upload \
         --user jmcfarlane \
         --repo notable \
@@ -38,6 +41,17 @@ for goos in darwin freebsd linux windows; do
         --name "notable-${TAG}.${goos}-amd64.zip" \
         --file target/notable-${TAG}.${goos}-amd64.zip
     echo "Uploaded: target/notable-${TAG}.${goos}-amd64.zip"
+done
+
+# Uploads for arm
+for v in ${arms[@]}; do
+    github-release upload \
+        --user jmcfarlane \
+        --repo notable \
+        --tag $TAG \
+        --name "notable-${TAG}.linux-arm${v}.zip" \
+        --file target/notable-${TAG}.linux-arm${v}.zip
+    echo "Uploaded: target/notable-${TAG}.linux-arm${v}.zip"
 done
 
 # Tag for release

